workos/event_objects/directory_group.py

```diff
@@ -0,0 +1,141 @@
+from typing import Literal, TypedDict
+from workos.utils.types import JsonDict
+
+
+class DirectoryGroupEvent(TypedDict):
+    id: str
+    name: str
+    idp_id: str
+    directory_id: str
+    organization_id: str
+    created_at: str
+    updated_at: str
+    raw_attributes: JsonDict
+    previous_attributes: JsonDict
+    object: Literal["directory_group"]
+
+
+class WorkOSDirectoryGroupEvent:
```

Comment:
Suggest rename to avoid confusion between data/payload and the actual event class.
```suggestion
class WorkOSDirectoryGroupEventData:
```